examples/pytorch/mnist/README.md

```diff
@@ -27,6 +29,13 @@ pip install dlrover -U
 
 Then, we can use `dlrover-run` to start the training by
 
+```bash
+dlrover-run --standalone --nproc_per_node=${GPU_NUM} \
```

Comment:
Now, we don't need to set `--standalone` to run locally. 
`dlrover-run --nproc_per_node=${GPU_NUM}` is OK.